autoload/clap/preview.vim

```diff
@@ -93,10 +106,10 @@ endfunction
 " the origin line should be positioned.
 " 0-based
 function! clap#preview#get_line_range(origin_lnum, range_size) abort
-  if a:origin_lnum - a:range_size > 0
+  if a:origin_lnum > a:range_size
     return [a:origin_lnum - a:range_size, a:origin_lnum + a:range_size, a:range_size]
   else
-    return [0, a:origin_lnum + a:range_size, a:origin_lnum]
+    return [0, a:range_size * 2, a:range_size]
```

Comment:
What problem this change resolves?

Reply:
This will make the float windows size `fixed` to preview_size instead of change to a smaller size if the search result is e.g. at 1st line.